pkg/endpoints/endpoints.go

```diff
@@ -57,7 +57,7 @@ var (
 // ParseServiceEndpointFlag parses the command line flag of service endpoint in the format ${ServiceRegion}:${ServiceID1}=${URL1},${ServiceID2}=${URL2...}
 // returning a list of ServiceEndpoint.
 func ParseServiceEndpointFlag(serviceEndpoints string) ([]ServiceEndpoint, error) {
-	if serviceEndpoints == "" {
+	if serviceEndpoints == "" || serviceEndpoints == "\"\"" {
```

Comment:
This will be `""` instead of empty string which is not great idea, I prefer setting some minimal value fed or think about some other way to fix this!

Reply:
Fixed - default value `none`